performance-storage-service/pss_project/api/models/database/OLTPBenchResult.py

```diff
@@ -14,6 +14,16 @@ class Meta:
         (SIMPLE_MODE, 'simple'),
         (EXTENDED_MODE, 'extended'),
     ]
+    RAM_DISK = 'RAM disk'
+    HDD = 'HDD'
+    SATA_SSD = 'SATA SSD'
+    NVME_SSD = 'NVMe SSD'
+    WAL_DEVICE_CHOICES = [
+        (RAM_DISK, 'RAM disk'),
+        (HDD, 'HDD'),
+        (SATA_SSD, 'SATA SSD'),
+        (NVME_SSD, 'NVMe SSD')
+    ]
```

Comment:
I think we also need an 'Unknown' as default.

Reply:
I agreed with Dan's comments on Slack: "'Since this is a user specified value it should never have an ‘unknown’ value. We should throw an error if the user forgets to specify"